packages/client/src/pages/Profile/elements/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,43 @@
+import { AvatarImage } from '@assets/images'
+import styles from './avatar.module.css'
+import { ChangeEvent, FC, useState } from 'react'
+import clsx from 'clsx'
+import { AvatarProps } from './type'
+import { updateAvatar } from '@services/profile'
+
+export const Avatar: FC<AvatarProps> = ({ avatar }) => {
+  const [avatarProfile, setAvatar] = useState<string>(avatar ? avatar : '')
+
+  const handleFileChange = async (e: ChangeEvent<HTMLInputElement>) => {
+    const file = e.target.files?.[0]
+    if (file) {
+      const formData = new FormData()
+      formData.append('avatar', file)
+      const newAvatar = await updateAvatar(formData)
+      setAvatar(newAvatar)
+    }
+  }
+
+  return (
+    <div
+      className={clsx(
+        styles.avatarContainer,
+        !avatarProfile
+          ? styles.avatarBackgroundDefault
+          : styles.avatarBackground
```

Comment:
[Исправление]
Условия с отрицанием читаются сложнее, тут лучше от него избавиться
```suggestion
        avatarProfile
          ? styles.avatarBackground
          : styles.avatarBackgroundDefault
```